integration/env.go

```diff
@@ -24,18 +25,40 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cmdrunner/step"
 	"github.com/ignite/cli/ignite/pkg/cosmosfaucet"
 	"github.com/ignite/cli/ignite/pkg/gocmd"
+	"github.com/ignite/cli/ignite/pkg/gomodulepath"
 	"github.com/ignite/cli/ignite/pkg/httpstatuschecker"
 	"github.com/ignite/cli/ignite/pkg/xexec"
 	"github.com/ignite/cli/ignite/pkg/xurl"
 )
 
 const (
 	ServeTimeout = time.Minute * 15
-	IgniteApp    = "ignite"
 	ConfigYML    = "config.yml"
 )
 
-var isCI, _ = strconv.ParseBool(os.Getenv("CI"))
+var (
+	isCI, _   = strconv.ParseBool(os.Getenv("CI"))
+	IgniteApp string
+)
+
+func init() {
+	wd, _ := os.Getwd()
+	_, appPath, err := gomodulepath.Find(wd)
+	if err != nil {
+		panic(err)
+	}
+	// Build the ignite binary
+	tmp, err := os.MkdirTemp("", "integration-bin")
+	if err != nil {
+		panic(err)
+	}
+	IgniteApp = path.Join(tmp, "ignite")
+	command := exec.Command("go", "build",
+		"-o", IgniteApp, appPath+"/ignite/cmd/ignite")
```

Comment:
Can we define this path in a const defined in the beginning of the file?

Reply:
sure [e8dcbdb](https://github.com/ignite/cli/pull/2639/commits/e8dcbdb3b2d2f31b74747618852893eebf52930b)